packages/meteor-accounts-saml/saml_server.js

```diff
@@ -182,61 +186,64 @@ middleware = function (req, res, next) {
 			return;
 		}
 
-		if (!samlObject.actionName)
-			throw new Error("Missing SAML action");
+		if (!samlObject.actionName) {
+			throw new Error('Missing SAML action');
+		}
 
-		console.log(Accounts.saml.settings.providers)
-		console.log(samlObject.serviceName)
+		console.log(Accounts.saml.settings.providers);
+		console.log(samlObject.serviceName);
 		var service = _.find(Accounts.saml.settings.providers, function (samlSetting) {
 			return samlSetting.provider === samlObject.serviceName;
 		});
 
 		// Skip everything if there's no service set by the saml middleware
-		if (!service)
-			throw new Error("Unexpected SAML service " + samlObject.serviceName);
+		if (!service) {
+			throw new Error('Unexpected SAML service ' + samlObject.serviceName);
+		}
+		var _saml;
 		switch (samlObject.actionName) {
-		case "metadata":
+		case 'metadata':
 			_saml = new SAML(service);
-			service.callbackUrl = Meteor.absoluteUrl("_saml/validate/" + service.provider);
+			service.callbackUrl = Meteor.absoluteUrl('_saml/validate/' + service.provider);
 			res.writeHead(200);
 			res.write(_saml.generateServiceProviderMetadata(service.callbackUrl));
 			res.end();
 			//closePopup(res);
 			break;
-		case "logout":
+		case 'logout':
 			// This is where we receive SAML LogoutResponse
 			_saml = new SAML(service);
 			_saml.validateLogoutResponse(req.query.SAMLResponse, function (err, result) {
 				if (!err) {
 					var logOutUser = function (inResponseTo) {
 						if (Accounts.saml.settings.debug) {
-						console.log("Logging Out user via inResponseTo " + inResponseTo);
+						console.log('Logging Out user via inResponseTo ' + inResponseTo);
 						}
 						var loggedOutUser = Meteor.users.find({
 							'services.saml.inResponseTo': inResponseTo
 						}).fetch();
-						if (loggedOutUser.length == 1) {
+						if (loggedOutUser.length === 1) {
 							if (Accounts.saml.settings.debug) {
-							console.log("Found user " + loggedOutUser[0]._id);
+							console.log('Found user ' + loggedOutUser[0]._id);
 							}
 							Meteor.users.update({
 								_id: loggedOutUser[0]._id
 							}, {
 								$set: {
-									"services.resume.loginTokens": []
+									'services.resume.loginTokens': []
 								}
 							});
 							Meteor.users.update({
 								_id: loggedOutUser[0]._id
 							}, {
 								$unset: {
-									"services.saml": ""
+									'services.saml': ''
 								}
 							});
 						} else {
-							throw new Meteor.error("Found multiple users matching SAML inResponseTo fields");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Meteor" is not defined.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2313371378/issues/source?bid=3199634&fileBranchId=3199636#l244)